accounts/tests/tests.py

```diff
@@ -0,0 +1,23 @@
+from django.template.loader import get_template
+
```

Comment:
unnecessary empty line

Reply:
Why is it unnecessary? 2 lines between imports and function/class is required

Reply:
![image](https://user-images.githubusercontent.com/93659937/165702035-33362570-908c-4c11-95cb-93469f19de98.png)
line 2

Reply:
hoo, that because it two different types of imports, one import from the project files and one from 3rd party library

Reply:
ok